### PR TITLE
Added ShipProfile

### PR DESCRIPTION
**Pull request by Asalz20**
ShipProfile was built with Api functionality for the SpaceX API

### CODE REVIEW

**Comment by ControlAltTea** [comment]
booyah

**Review by FullStackCorm — COMMENTED** [review]
all good fam

File: public/js/carousel.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+let slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+//
```
Comment (HermanDhillon) [inline public/js/carousel.js]: We dont need this comment
```suggestion
```

**Review by HermanDhillon — CHANGES_REQUESTED** [review]
left a comment in carousel.js

**Review by HermanDhillon — APPROVED** [review]
exceptional work alex

**Pull request merged**